src/main/resources/lang/dutch.lang

```diff
@@ -0,0 +1,190 @@
+	finished loading: Klaar met laden.
+
+# -- Skript command --
+skript command:
+	usage: Usage:
+	help:
+		description: Skript's hoofdcommando
+		help: Drukt deze helpmelding af. Gebruik '/skript reload/enable/disable/update' voor meer informatie.
+		reload:
+			description: Herlaadt een specifiek script, alle scripts, de configuratie of alles
+			all: Herlaadt de configuratie, alle aliassenconfiguraties en alle scripts
+			config: Herlaadt de hoofdconfiguratie
+			aliases: Herlaadt de aliassen-configuratie (aliases-english.zip of plugin jar)
+			scripts: Herlaadt alle scripts
+			<script>: Herlaadt een specifiek script of een folder met scripts
+		enable:
+			description: Schakelt alle scripts of een specifiek script in
+			all: Schakelt alle scripts in
+			<script>: Schakelt een specifiek script of een map met scripts in
+		disable:
+			description: Schakelt alle scripts of een specifiek script uit
+			all: Schakelt alle scripts uit
+			<script>: Schakelt een specifiek script of een folder met scripts uit
+		update:
+			description: Controleer of er updates zijn, lees de lijst met wijzigingen of download de nieuwste versie van Skript
+			check: Controleert op een nieuwe versie
+			changes: Toont alle wijzigingen sinds de huidige versie
+			download: Download de nieuwste versie
+		info: Drukt een bericht af met koppelingen naar de aliassen en documentatie van Skript
+		gen-docs: Genereert documentatie met behulp van docs/templates in de plugin folder
+		test: Gebruikt voor het uitvoeren van interne Skript-tests
+
+	invalid script: Kan het script <grey>'<gold>%s<gold>'<red> niet in de scripts folder vinden!
+	invalid folder: Kan de folder <grey>'<gold>%s<gold>'<red> niet in de scripts folder vinden!
+	reload:
+		warning line info: <gold><bold>Lijn %s:<gray> (%s)<reset>\n
+		error line info: <light red><bold>Lijn%s:<gray> (%s)<reset>\n
+		reloading: <gold>%s<reset> herladen...
+		reloaded: <lime>Succesvol herladen <gold>%s<lime>. <gray>(<gold>%2$sms<gray>)
+		error: <gold>%2$s <light red>error¦¦s¦ <light red>tegengekomen tijdens het herladen <gold>%1$s<light red>! <gray>(<gold>%3$sms<gray>)
+		script disabled: <gold>%s<reset> is momenteel uitgeschakeld. Gebruik <gray>/<gold>skript <cyan>enable <red>%s<reset> om het in te schakelen.
+		warning details: <yellow>    %s<reset>\n
+		error details: <light red>    %s<reset>\n
+		other details: <white>    %s<reset>\n
+		line details: <gold>    Lijn: <gray>%s<reset>\n <reset>
+
+		config, aliases and scripts: de configuratie, aliassen en alle scripts
+		scripts: alle scripts
+		main config: de hoofdconfiguratie
+		aliases: de aliassen
+		script: <gold>%s<reset>
+		scripts in folder: alle scripts in <gold>%s<reset>
+		x scripts in folder: <gold>%2$s <lime>script¦¦s¦ in <gold>%1$s<reset>
+		empty folder: <gold>%s<reset> bevat geen ingeschakelde scripts.
+	enable:
+		all:
+			enabling: Alle uitgeschakelde scripts inschakelen...
+			enabled: Alle eerder uitgeschakelde scripts ingeschakeld en geparseerd
```

Comment:
```suggestion
			enabled: Alle eerder uitgeschakelde scripts zijn ingeschakeld en geparsed
```

to parse is afaik not translatable in this context. even if it is, i'm not sure if it would add anything